app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -99,6 +99,18 @@ class PlatformInitiativesService @Inject()(
         version               = Version(2,13,0,"2.13.0"),
         team                  = team,
         experimental          = true
+      ),
+      createMigrationInitiative(
+        initiativeName        = "Depreciate Simple-Reactivemongo: HMRC-Mongo Migration",
```

Comment:
`Deprecate`? Or simply "Replace simple-reactivemongo with hmrc-mongo"?

Reply:
Whoops! I'll change that to the latter.